common/game_rules/00_game_rules.txt

```diff
@@ -131,42 +107,46 @@ monument_effects = {
 		flag = disable_pm_default_building_taj_mahal
 		flag = disable_pm_default_building_vatican_city
 		flag = disable_pm_default_building_white_house
-			
 		flag = disable_pm_monument_prestige_only
 		flag = disable_pm_monument_prestige_only_vatican_city
-		
 		flag = force_pm_monument_no_effects
 	}
 }
 
 subject_flags = {
 	default = allow_subject_flags
-
 	no_subject_flags = {
 		flag = no_subject_flags
 	}
-
 	allow_subject_flags = {
 	}
 }
 
 subject_map_color = {
 	default = allow_subject_map_color
-
 	no_subject_map_color = {
 		flag = no_subject_map_color
 	}
-
 	allow_subject_map_color = {
 	}
 }
 
 customs_union_diplo_plays = {
 	default = enable_customs_union_diplo_plays
-
 	enable_customs_union_diplo_plays = {
 	}
-	
 	disable_customs_union_diplo_plays = {
 	}
-}
\ No newline at end of file
+}
+
+player_construction_bonus_value = {
+	default = enable_player_construction_bonus_15
```

Comment:
and default should be single player based

Reply:
Like so?

Reply:
yes